wled00/ir.cpp

```diff
@@ -618,7 +618,8 @@ void decodeIRJson(uint32_t code)
         cmdStr = "win&" + cmdStr;
       }
       handleSet(nullptr, cmdStr, false); 
-    }        
+    }
+    colorUpdated(CALL_MODE_BUTTON);
```

Comment:
Doesn't this belong to a line above closing brace? Immediately after handleSet().